.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,16 +1,36 @@
-<!--
-Description of PR
--->
 
-<!--
+This PR has the objective of...
+
+## Changes
+
+1. Added new `xyz` to...
+2. Fixed a bug in...
+3. Changed the behaviour of...
+4. Improved the docs about...
+
+## Checklist
+
 Checklist for for pull requests including new code and/or changes to existing code...
--->
-
-* [ ] Includes unit tests
-* [ ] New functions have docstrings with examples that can be run with doctest
-* [ ] New functions are included in API docs
-* [ ] Docstrings include notes for any changes to API or behaviour
-* [ ] Travis CI passes (unit tests run under Linux)
-* [ ] AppVeyor CI passes (unit tests run under Windows)
-* [ ] Unit test coverage has not decreased (see Coveralls)
-* [ ] All changes documented in docs/changes.rst
+
+* [ ] Code
+  * [ ] Includes unit tests
+  * [ ] New functions have docstrings with examples that can be run with doctest
+  * [ ] New functions are included in API docs
+  * [ ] Docstrings include notes for any changes to API or behaviour
+  * [ ] All changes documented in docs/changes.rst
+* [ ] Testing
+  * [ ] \(Optional) Tested local against remote servers
+  * [ ] Travis CI passes (unit tests run under Linux)
+  * [ ] AppVeyor CI passes (unit tests run under Windows)
+  * [ ] Unit test coverage has not decreased (see Coveralls)
+* [ ] Changes
+  * [ ] \(Optional) Just a proof of concept
+  * [ ] \(Optional) Work in progress
+  * [ ] Ready to review
+  * [ ] Ready to merge
+
+### Reference
+
+* [ ] Not done yet
+* [x] Finished
+* [x] ~~Doesn't apply to this PR~~
```

Comment:
Hi @juarezr, sorry I didn't get it first time, now I see what is meant by the "Legend" section :-)

FWIW this section was a bit confusing to me, I thought these were more check-boxes the PR submitter was supposed to complete. Hopefully most PR submitters will understand how to use checkboxes and do something sensible if a line does not apply to the PR, suggest you could remove this section.

Reply:
I agree.